.github/workflows/_build_test.yml

```diff
@@ -30,19 +29,42 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - run: ./build.sh ${{ matrix.component }}
+      - name: Set up QEMU
```

Comment:
This is not necessary at the moment, but takes ~2 seconds to boot and leaves us in a place to test more with multi-platform. Or we comment it out or remove it, adding a note to the multi-platform doc...